### ISSUE TITLE
mixed-indentation does not catch indent errors on multiline array/object

### ISSUE THREAD

**Issue by maximecharron** [PyCQA/pylint#2984]
### Steps to reproduce
1. Have a mix of spaces and tabs in a multiline object/array
```
// Spaces everywhere for the indentation
foo = {
    'bar': [
        'foobar',
		'barfoo', // Tabs on this line instead of spaces
    ]
}
```
2. Run pylint
3. Notice it doesn't catch the mixed-indentation

### Current behavior
Pylint doesn't catch the mixed indentation (see code snippet in steps to reproduce)

### Expected behavior
Pylint should catch the mixed indentation and report an error

### pylint --version output
```
Using config file foo\.pylintrc
__main__.py 1.9.2,
astroid 1.6.5
Python 3.6.5 (v3.6.5:f59c0932b4, Mar 28 2018, 17:00:18) [MSC v.1900 64 bit (AMD64)]
```

**Comment by PCManticore** [comment]
Thanks for the report.

**Comment by Pierre-Sassoulas** [comment]
`mixed-indentation` has been removed in #3578 , it is no longer useful since TabError is included directly in python3
